### DISCUSSION TITLE
best practice question: NIC Config to receive syslog for Standalone install?

### DISCUSSION THREAD

**Discussion by ionTomorrow** [Security-Onion-Solutions/securityonion#2458] [Unsupported Versions]
Hi,

I was using the earlier of SO for a long time, but have added a new install for v2.3.1 I read here: https://github.com/Security-Onion-Solutions/securityonion/discussions/2445 that the management interface is no longer being monitored.

I used the old IP of my SO box for the new one, so all the syslog devices are sending to the same IP. However I no longer am seeing any syslog collected from the new box. I have added the entire subnet for syslog using so-allow, but nothing seems to come through.

I do have a second NIC setup for sniffing, however that doesn't have an IP.

Would the recommended config for collecting syslog be to continue to send to the management IP I have set (and figure out my problem), or add a 3rd NIC just for syslog?

**Comment by TOoSmOotH** [comment]
You need to run soup. Syslog should work out of the box in later versions. Having a dedicated NIC here won't solve the issue. Try the soup and let me know if syslog starts showing up.
